end_to_end_tests/golden-record-custom/custom_e2e/models/a_model.py

```diff
@@ -128,6 +189,24 @@ def _parse_a_camel_date_time(data: Any) -> Union[datetime.datetime, datetime.dat
 
         required_not_nullable = d.pop("required_not_nullable")
 
+        def _parse_one_of_models(data: Dict[str, Any]) -> Union[FreeFormModel, ModelWithUnionProperty]:
+            one_of_models: Union[FreeFormModel, ModelWithUnionProperty]
+            try:
+                if not isinstance(data, dict):
+                    raise TypeError()
+                one_of_models = FreeFormModel.from_dict(data)
```

Comment:
Yeah, that's a good point. Actually initially I didn't have a runtime check, but then MyPy throws an error, because it's not type-safe.

I'm starting to think that maybe the original `Any` annotation is better, since we can't be confident in the data passed in. But I think it should be `object`, not `Any`, because `Any` causes type errors to be ignored, whereas `object` will force us to be type-safe.

Reply:
Sounds good regarding `object` vs `Any` if that ends up working better.